src/app/components/Transcript/index.styles.tsx

```diff
@@ -0,0 +1,64 @@
+import { css, Theme } from '@emotion/react';
+
+export default {
+  transcript: ({ spacings, palette, isDarkUi }: Theme) =>
+    css({
+      backgroundColor: isDarkUi ? palette.GREY_7 : palette.WHITE,
+      display: 'block',
+      marginBottom: `${spacings.TRIPLE}rem`,
+      padding: `${spacings.DOUBLE}rem`,
+      color: palette.GREY_6,
```

Comment:
Could we change the colour to the same as what we use for image caption text please?